### ISSUE TITLE
Direct EXE downlod

### ISSUE THREAD

**Issue by And96** [caesiumstudio/csBooks-updates#10]
Request:

Allow to download EXE for those who dont have Microsoft Store.

**Comment by caesiumstudio** [comment]
csBooks is only published for Window 10 version and Windows 10 should have MS store. 
Could you please provide more details about why you don't have Microsoft Store.

**Comment by And96** [comment]
Not all W10 versions have store included.

e.g: LTSB/LTSC (Long-Term Servicing Channel) does NOT have store.

And not everyone has a MS account.

IMHO and EXE is an easy way.

**Comment by wadbr** [comment]
Agreed, please provide non microsoft store options. I do not have or use a microsoft email address so I dont have access to the store.

**Comment by AhmedThahir** [comment]
Yes please

**Comment by guategeek** [comment]
I also would like a MS download from here on github, rather frustrating that you have packages for every other OS on here.

**Comment by helpppeerr** [comment]
https://drive.google.com/file/d/1sN4c8Ym04jIdSPwIleuZ1z-sAShbY9tL/view?usp=sharing

**Comment by helpppeerr** [comment]
> I also would like a MS download from here on github, rather frustrating that you have packages for every other OS on here.

https://drive.google.com/file/d/1sN4c8Ym04jIdSPwIleuZ1z-sAShbY9tL/view?usp=sharing

> I also would like a MS download from here on github, rather frustrating that you have packages for every other OS on here.



> Not all W10 versions have store included.
> 
> e.g: LTSB/LTSC (Long-Term Servicing Channel) does NOT have store.
> 
> And not everyone has a MS account.
> 
> IMHO and EXE is an easy way.

https://drive.google.com/file/d/1sN4c8Ym04jIdSPwIleuZ1z-sAShbY9tL/view?usp=sharing

**Comment by helpppeerr** [comment]
> Not all W10 versions have store included.
> 
> e.g: LTSB/LTSC (Long-Term Servicing Channel) does NOT have store.
> 
> And not everyone has a MS account.
> 
> IMHO and EXE is an easy way.



> Yes please



> I also would like a MS download from here on github, rather frustrating that you have packages for every other OS on here.


Guys i found exe. It is in "app" file. https://drive.google.com/file/d/1sN4c8Ym04jIdSPwIleuZ1z-sAShbY9tL/view?usp=sharing

**Comment by caesiumstudio** [comment]
Be aware that there is no official EXE file provided by Caesium Studio right now. We are working on that and would provide an EXE download soon. Anybody offering an EXE until then can be providing a potential virus that threatens your security.

**Comment by charisme** [comment]
Another vote for this. I'm on LTSC so I can't have MS Store so I'll be waiting for the official EXE. Thanks in advance

**Comment by generalxid** [comment]
yes upload it as soon as possible.

**Comment by And96** [comment]
Can anyone upload a compiled version?

**Comment by michael1maass** [comment]
Hi, I also do support the request to provide an installable file for Windows. Be it an .msi or an .exe file. Especially in corporate environments, Microsoft App Store is not an Option. 
Thank you
